src/runtime/LoadingView.tsx

```diff
@@ -17,31 +18,39 @@ export default class LoadingView extends React.Component<Props> {
       const error = value["_error"].message;
 
       // make sure the error isn't a duplicate
-      const found = this.errors.findIndex((err) => 
-        err === error
-      ) > -1;
+      const found = this.errors.indexOf(error) > -1;
 
       if (!found) {
         this.errors.push(error);
       }
     }
-  }
+  };
 
   public render() {
     const { logs } = this.props;
 
     R.forEachObjIndexed(this.findErrorKeys, logs);
     return (
-      <Popup
-        trigger={<Spinner name='double-bounce'/>}
-        position="right center"
-        on="hover"
-        contentStyle={{ width: "auto", display: "flex", flexWrap: "wrap" }}
-      >
-        <div>
-          {this.errors.map(error => error)}
-        </div>
-      </Popup>
-    )
+      <Loader.Render>
+        {(CustomLoader: React.ComponentProps<any>) =>
```

Comment:
Please use lowercase for local variables `customLoader`

Also please change the type to be `((props: RenderProps) => ReactElement) | undefined`. We probably want to typedef for that function like `RenderFunc`.

Reply:
I was using `CustomLoader` because it was a component - has been fixed now!